### PR TITLE
feat: added all types of cards

### PR DESCRIPTION
**Pull request by rishbitsnbytes**
Please review!

Preview is below :

ezgif com-gif-maker

### CODE REVIEW

File: Components/cards.css [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    border: none;
+    font-weight: bold;
+    font-size: small;
+    border-radius: 0.5rem;
+    padding: 0.5rem 0.3rem;
+}
+
+.card-button:hover, .card-icon:hover { 
+    background: rgba(79, 189, 186, 0.2)
+}
```
Comment (ravikumar0403) [inline Components/cards.css]: use variable for this color as well

File: Components/cards.css [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+.card-icon {
+    margin: 0 0.1rem;
+    cursor: pointer;
+    color: var(--primary-color);border-radius: 0.5rem;
```
Comment (ravikumar0403) [inline Components/cards.css]: check the formatting for this line

**Review by ravikumar0403 — APPROVED** [review]
Code and design looks good. I've added a couple of sanity comments.

File: Components/cards.css [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+/* <!-- Card with Badge --> */
+
+.card-badge {
+    position: absolute;
+    top: 1rem;
+    box-sizing: inherit;
+    border-radius: 0.5rem;
+    background-color: red;
+    color: white;
+    padding: 0.2rem 1rem;
```
Comment (sonishreyas) [inline Components/cards.css]: you can use variables for padding and margins as well if you want it really helped me.

File: Components/cards.html [hunk 4]
```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (sonishreyas) [inline Components/cards.html]: You can add a meta author tag here

File: Components/cards.html [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+            <i class="card-icon fa-regular     fa-heart"></i>
+            <i class="card-icon fa-solid fa-share-nodes"></i>
+            <i class="card-icon fa-solid fa-ellipsis-vertical"></i>
+        </div>
+    </div>
+    <button class="card-dismiss-button card-icon card-button"><i class="fa-regular fa-circle-xmark"></i></button>
+    </div>
+    </div>
+
+
```
Comment (sonishreyas) [inline Components/cards.html]: you can remove the unwanted space if you don't need it

**Review by sonishreyas — APPROVED** [review]
I have kept a few suggestions you can implement. Rest it looks good to me

File: Components/cards.css [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+    width: 100%;
+    height: 100%;
+    border-radius: 0.5rem
+}
+
+.card-overlay-text {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+    position: absolute;
```
Comment (piyushsingh0992) [inline Components/cards.css]: Use Flex for centering a components

File: Components/cards.html [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+        <h3 class="card-title">Artistic Weapon</h3>
+        <h5 class="card-subtitle">Indian Mother's/Gf's Brahmastra</h5>
+        <p class="card-para">Will hit best when aimed for head with pointed end.</p>
+        <div class="card-actions-container">
+            <div class="card-buttons-container">
+                <button class="card-button">Buy</button>
+                <button class="card-button">Add to Cart</button>
+            </div>
+            <div class="card-icons-conatainer">
+                <i class="card-icon fa-regular     fa-heart"></i>
```
Comment (piyushsingh0992) [inline Components/cards.html]: Remove extra space.

**Comment by rishbitsnbytes** [comment]
Thanks guys!!
All the changes have been refactored.

**Pull request merged**